utils/resolve.js

```diff
@@ -80,19 +80,20 @@ exports.fileExistsWithCaseSync = function fileExistsWithCaseSync(filepath, cache
   return result;
 };
 
-function relative(modulePath, sourceFile, settings) {
-  return fullResolve(modulePath, sourceFile, settings).path;
+function relative(modulePath, sourceFile, context) {
```

Comment:
this is actually a breaking change for the utility package, so that seems quite suboptimal

Reply:
Can I resolve that by accepting either the new value (context) or the old (settings), as in the latest commit?